EventsExpress/ClientApp/src/components/footer/footer.css

```diff
@@ -0,0 +1,80 @@
+.custom-footer {
+    margin-top: 30px;
+    z-index: 500;
+    background-color: rgb(184, 184, 184);
+    position: relative;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    padding: 5px;
+}
+
+.custom-footer__links-to-pages {
```

Comment:
Please do not separate classes by `__`. In scss you can make classes be children of other classes - just use that feature.